console.rst

```diff
@@ -79,18 +79,37 @@ Configuring the Command
 You can optionally define a description, help message and the
 :doc:`input options and arguments </console/input>`::
 
-    // ...
-    protected function configure()
-    {
-        $this
-            // the short description shown while running "php bin/console list"
-            ->setDescription('Creates a new user.')
+.. versionadded:: 5.3
 
-            // the full command description shown when running the command with
-            // the "--help" option
-            ->setHelp('This command allows you to create a user...')
-        ;
-    }
+    The ability to use PHP attributes to configure commands was introduced in
+    Symfony 5.3.
+
+.. configuration-block::
+
+    .. code-block:: php-attributes
+
+            // ...
+            #[ConsoleCommand(
+                name: 'app:create-user',
+                description: 'Creates a new user.'
+                hidden: false,
+                aliases: [],
+            )]
+
+    .. code-block:: php
+
+        // ...
+        protected function configure()
+        {
+            $this
+                // the short description shown while running "php bin/console list"
+                ->setDescription('Creates a new user.')
+
+                // the full command description shown when running the command with
+                // the "--help" option
+                ->setHelp('This command allows you to create a user...')
```

Comment:
could you please add ``hidden`` and ``aliases`` in this example too?

`app::new-user` could be used as an alias in both examples.

Thanks